### DISCUSSION TITLE
Is it possible to change the camera model?

### DISCUSSION THREAD

**Discussion by nickponline** [NVlabs/instant-ngp#634] [Q&A]
Is it possible to change the camera type used by Instant NGP. I would like to train using a *projective camera* but render the results using an *orthographic camera*. The reason is I would like to try and compute an ortho-mosaic of a 3D model?

**Comment by Tom94** [comment]
__instant-ngp__ currently doesn't support an orthographic camera model, sorry. It'd have to be added to the pixel-to-ray, pos-to-pixel, and motion vector routines.
